games-action/tuxracer/patches/tuxracer-0.61.patch

```diff
@@ -0,0 +1,20 @@
+--- tuxracer-0.61/src/game_config.c.org	2016-04-25 18:59:37.147587072 +0200
++++ tuxracer-0.61/src/game_config.c	2016-04-25 18:59:23.122945536 +0200
+@@ -79,7 +79,7 @@
+ #  if defined( WIN32 )
+ #    define DATA_DIR "."
+ #  else
+-#    define DATA_DIR "/usr/local/share/tuxracer"
++#    define DATA_DIR "/boot/home/config/apps/TuxRacer/data"
```

Comment:
how about a patch you can upstream?  maybe try addin an "if defined( HAIKU
)" instead of breaking it on systems that use
/usr/local/share/putmyprogramhere

-scott

On Fri, Apr 29, 2016 at 7:39 AM, Begasus <notifications@github.com> wrote:

> In games-action/tuxracer/patches/tuxracer-0.61.patch
> <https://github.com/haikuports/haikuports/pull/572#discussion_r61587724>:
>
> > @@ -0,0 +1,20 @@
> > +--- tuxracer-0.61/src/game_config.c.org	2016-04-25 18:59:37.147587072 +0200
> > ++++ tuxracer-0.61/src/game_config.c	2016-04-25 18:59:23.122945536 +0200
> > +@@ -79,7 +79,7 @@
> > + #  if defined( WIN32 )
> > + #    define DATA_DIR "."
> > + #  else
> > +-#    define DATA_DIR "/usr/local/share/tuxracer"
> > ++#    define DATA_DIR "/boot/home/config/apps/TuxRacer/data"
>
> Not familiar with macros, but with the latest changes you can even install
> them in /boot/system/packages ...
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/haikuports/haikuports/pull/572/files/f321d132c4644e4f5a881bff20bd8651d0aabfc9#r61587724>
>